types/ember/test/ember-module-tests.ts

```diff
@@ -2,8 +2,12 @@ import Ember from 'ember';
 
 // $
 Ember.$; // $ExpectType JQueryStatic
+
+const top = (<T>(x?: T): T => x!)();
+type Top = typeof top;
+declare function expectTypeNativeArrayTop(x: Ember.NativeArray<Top>): void;
 // A
-Ember.A(); // $ExpectType NativeArray<{}>
+expectTypeNativeArrayTop(Ember.A());
```

Comment:
@weswigham - Isn't this a less-strict test than what it replaces? i.e., if a regression were introduced where `Ember.A()`'s return type were inappropriately widened to `any`, the old test would have caught it and the new one wouldn't.

Reply:
Happy to fix this myself in a follow-on PR btw. I just don't know if loosening the test was part of the adjustment necessary to work well with Microsoft/TypeScript#30637

Reply:
The loosening isn't required for anything - I was just trying to keep the updated tests simple, esp. since I believe the only way to do the exact test is via conditional types, which not every package in the PR can use (due to being targeted at older TS versions).